pyxem/utils/indexation_utils.py

```diff
@@ -71,6 +71,121 @@ def fast_correlation(image_intensities, int_local, pn_local, **kwargs):
     )  # Correlation is the partially normalized dot product
 
 
+def sum_absolute_differences(image_intensities, int_local, **kwargs):
+    """
+    Computes the correlation score between an image and a template, using the formula
+    .. math:: sum_absolute_differences
+        \\\\sum_{j=1}^m P(x_j, y_j) - T(x_j, y_j)
+
+    Parameters
+    ----------
+    image_intensities: list
+        list of intensity values in the image, for pixels where the template has a non-zero intensity
+    int_local: list
+        list of all non-zero intensities in the template
+
+    Returns
+    -------
+    corr_local: float
+        correlation score between template and image.
+
+    See also:
+    ---------
+    correlate_library, fast_correlation, normalized_sum_squared_differences
+
+    """
+    return (
+        np.sum(np.subtract(np.abs(image_intensities), np.abs(int_local)))
```

Comment:
@norskie is this right? Isn't the absolute difference between `X` and `Y` this: `np.sum(np.abs(X - Y))`?